AWS EFS backup (#2014)
giantswarm/docs: src/content/advanced/storage/efs/_index.md

---
linkTitle: Amazon Elastic File System
title: Amazon Elastic File System (EFS)

> description: AWS EFS is a managed file storage offered by AWS

Author: ```suggestion
description: The Elastic File System is a managed file storage service offered by AWS
```

> description: The Elastic File System is managed file storage offered by AWS

Reviewer: ```suggestion
description: AWS EFS is a managed file storage offered by AWS
```

> description: The Elastic File System is managed file storage offered by AWS

Author: ```suggestion
description: The Elastic File System is a managed file storage service offered by AWS
```

> description: The Elastic File System is a managed file storage service offered by AWS

Author: ```suggestion
description: The Elastic File System is a managed file storage service offered by AWS.
```
